Cryptocurrency/coinmarketcap.30m.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/local/bin/python3
+
+#  <xbar.title>CoinMarketCap</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Alvaro Serrano</xbar.author>
+#  <xbar.author.github>https://github.com/alvaroserrrano</xbar.author.github>
+#  <xbar.desc>Your preferred cryptos in the Fiat currency you choose</xbar.desc>
+#  <xbar.image>https://ibb.co/DYFkPPP</xbar.image>
```

Comment:
please link to the actual image file, https://i.ibb.co/nByfqqq/Screen-Shot-2021-08-01-at-2-24-50-PM.png

Reply:
Thanks for letting me know! Just committed the change